code/game/objects/items/toy_mechs.dm

```diff
@@ -0,0 +1,630 @@
+			victim.combat_health-=2
+			playsound(src, 'sound/weapons/marauder.ogg', 20, TRUE)
+		if(SPECIAL_ATTACK_HEAL) //+2 healing
+			combat_health+=2
+			playsound(src, 'sound/mecha/mech_shield_raise.ogg', 20, TRUE)
+		if(SPECIAL_ATTACK_UTILITY) //+1 heal, +1 damage
+			victim.combat_health--
+			combat_health++
+			playsound(src, 'sound/mecha/mechmove01.ogg', 30, TRUE)
+		if(SPECIAL_ATTACK_OTHER) //other
+			super_special_attack(victim)
+		else
+			say("I FORGOT MY SPECIAL ATTACK...")
+
+/**
+  * Base proc for 'other' special attack moves. 
+  *
+  * This one is only for inheritance, each mech with an 'other' type move has their procs below. 
+  * Arguments:
+  * * victim - the toy being hit by the super special move (doesn't necessarily need to be used)
+  */
+/obj/item/toy/prize/proc/super_special_attack(obj/item/toy/prize/victim) 
+	visible_message("<span class='notice'> [src] does a cool flip.</span>")
+
+/obj/item/toy/prize/ripley
+	name = "toy Ripley"
+	desc = "1/13"
+	max_combat_health = 3
+	special_attack_type = SPECIAL_ATTACK_DAMAGE
+	special_attack_cry = "OMEGA DRILL"
+
+/obj/item/toy/prize/fireripley
+	name = "toy Firefighting Ripley"
+	desc = "2/13"
+	icon_state = "fireripleytoy"
+	max_combat_health = 4
+	special_attack_type = SPECIAL_ATTACK_UTILITY
+	special_attack_cry = "FIRE SHIELD"
+
+/obj/item/toy/prize/deathripley
+	name = "toy Deathsquad Ripley"
+	desc = "3/13"
+	icon_state = "deathripleytoy"
+	max_combat_health = 5
+	special_attack_type = SPECIAL_ATTACK_OTHER
+	special_attack_type_message = "instantly destroys the opposing mech if its health is less than this mech's health."
+	special_attack_cry = "KILLER CLAMP"
+
+/obj/item/toy/prize/deathripley/super_special_attack(obj/item/toy/prize/victim) 
+	playsound(src, 'sound/weapons/sonic_jackhammer.ogg', 20, TRUE)
+	if(victim.combat_health < combat_health) //Instantly kills the other mech if it's health is below our's.
+		say("EXECUTE!!")
+		victim.combat_health = 0
+	else //Otherwise, just deal one damage.
+		victim.combat_health--
+
+/obj/item/toy/prize/gygax
+	name = "toy Gygax"
+	desc = "4/13"
+	icon_state = "gygaxtoy"
+	max_combat_health = 5
+	special_attack_type = SPECIAL_ATTACK_UTILITY
+	special_attack_cry = "SUPER SERVOS"
+
+/obj/item/toy/prize/durand
+	name = "toy Durand"
+	desc = "5/13"
+	icon_state = "durandtoy"
+	max_combat_health = 6
+	special_attack_type = SPECIAL_ATTACK_HEAL
+	special_attack_cry = "SHIELD OF PROTECTION"
+
+/obj/item/toy/prize/honk
+	name = "toy H.O.N.K."
+	desc = "6/13"
+	icon_state = "honktoy"
+	max_combat_health = 4
+	special_attack_type = SPECIAL_ATTACK_OTHER
+	special_attack_type_message = "puts the opposing mech's special move on cooldown and heals this mech."
+	special_attack_cry = "MEGA HORN"
+
+/obj/item/toy/prize/honk/super_special_attack(obj/item/toy/prize/victim) 
+	playsound(src, 'sound/machines/honkbot_evil_laugh.ogg', 20, TRUE)
+	victim.special_attack_cooldown = 3 //Adds cooldown to the other mech and gives a minor self heal
```

Comment:
shouldnt this be +=?

Reply:
i didn't really intend on the cooldown ever going above 3, but i'll consider this change